dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -699,9 +704,9 @@ def _handle_deprecation(self, supported_arg, deprecated_arg, name):
                         "Both arguments were provided, but only `output_{name}` should be provided."
                     ).format(name=name))
                 # show only this warning
-                warnings.warn(("Use of argument `{name}` is deprecated. Please use `output_{name}` instead. " \
-                    + "`output_{name}` should be provided as a tuple or a single value.").format(name=name),
-                    Warning, stacklevel=2)
+                warnings.warn(("Use of argument `{name}` is deprecated. Please use `output_{name}`"
+                               " instead. `output_{name}` should be provided as a tuple"
+                               " or a single value.").format(name=name), Warning, stacklevel=2)
```

Comment:
May I ask, that you change that to `f""`?

Reply:
This is actually good as it is, since the single value of the argument is used multiple times.